SourceCode/AgOpenGPS.Core/ApplicationModels/BaseModels/GeoCoord.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+
+namespace AgOpenGPS.Core
+{
+    public class GeoCoord
+    {
+        private double _northing;
+        private double _easting;
+
+        public GeoCoord(double northing, double easting)
+        {
+            _northing = northing;
+            _easting = easting;
+        }
+
+        public double Northing => _northing;
+        public double Easting => _easting;
```

Comment:
Maybe use auto properties here:
```suggestion
        public double Northing { get; }
        public double Easting { get; }
```

Reply:
Yes, why not.